NOTICE

```diff
@@ -3,10 +3,3 @@ Copyright 2011-2016 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
-
-This product includes the following work from the Apache Thrift project:
-
-TNonblockingServerSocket.java
```

Comment:
I'm sure there's other ASF code in there, too... maybe from Hadoop, or HBase, or Cassandra. They are all covered.